docs/index.rst

```diff
@@ -3,18 +3,73 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Flax documentation
-==================
+Flax: Neural networks with JAX
+====
 
-Flax is a neural network library and ecosystem for JAX that is
-designed for flexibility. Flax is in use by a growing community of
-researchers and engineers at Google who happily use Flax for their
-daily research.
+Flax delivers an **end-to-end, flexible, user experience for researchers
+who use JAX with neural networks**. Flax exposes the full power of JAX.
+It is made up of loosely coupled libraries,
+which are showcased with end-to-end integrated guides and examples. |emphasized hyperlink|_
 
-For a quick introduction and short example snippets, see our `README
-<https://github.com/google/flax/blob/main/README.md>`_.
+.. |emphasized hyperlink| replace:: **Get started now ▶**
+.. _emphasized hyperlink: getting_started.html
+
+.. code:: python
+
+   class MLP(nn.Module):
```

Comment:
What about using an example that highlights Flax's strengths a bit better?

The challenge will be to keep it short and understandable. I think it's fine if the reader does not understand all the code, but they should be able to understand what is happening, and ideally be impressed by Flax's brevity and great flexibility to tackle difficult problems...